example-ideal-maze.txt

```diff
@@ -0,0 +1,21 @@
+# A picobot program that can solve a maze, using the right-hand rule
+
+moveUp:      # moving up
+	If can move right, move right and moveRight     # we can go right
```

Comment:
This line is actually kind of confusing because of the "can move right, move right and moveRight" its hard to tell which thing is which and what each one is doing. I'm not sure how you would change the syntax to make it more readable though.